src/lib.rs

```diff
@@ -28,11 +28,18 @@ pub mod block;
 pub mod dict;
 pub mod stream;
 
+use std::fmt::Debug;
 use std::io;
+use std::ops::RangeBounds;
 
 /// Default compression level.
 pub use zstd_safe::CLEVEL_DEFAULT as DEFAULT_COMPRESSION_LEVEL;
 
+/// The accepted range of compression levels.
+pub fn compression_level_range() -> impl RangeBounds<zstd_safe::CompressionLevel> + Debug {
```

Comment:
Any reason not to just return a [`RangeInclusive`](https://doc.rust-lang.org/beta/std/ops/struct.RangeInclusive.html)?

Reply:
Not really.